server/src/main/java/org/elasticsearch/client/internal/IndicesAdminClient.java

```diff
@@ -150,27 +147,6 @@ public interface IndicesAdminClient extends ElasticsearchClient {
      */
     IndicesSegmentsRequestBuilder prepareSegments(String... indices);
 
-    /**
-     * The shard stores info of one or more indices.
-     *
-     * @param request The indices shard stores request
-     * @return The result future
-     */
-    ActionFuture<IndicesShardStoresResponse> shardStores(IndicesShardStoresRequest request);
```

Comment:
Oh, I see it is removed now. Is it because it is localOnly now?

Reply:
No, it's really because these "helpful" client interfaces are not really useful now that the transport client is dead, and they are a bit of a dependency nightmare - they prevent moving the corresponding actions out into their own modules for instance. They also only work for actions defined in `server` anyway.